### PR TITLE
Generate POJOs during build. Remove associated tests and generation s…

### PR DESCRIPTION
**Pull request by bfink13**
…cript

### CODE REVIEW

File: src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-import com.fasterxml.jackson.annotation.JsonAnySetter;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonInclude;
-import org.apache.commons.lang.builder.EqualsBuilder;
-import org.apache.commons.lang.builder.HashCodeBuilder;
-import org.apache.commons.lang.builder.ToStringBuilder;
-
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@Generated("org.jsonschema2pojo")
-public class Value {
```
Comment (bfink13) [inline src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java:17]: This class is mysteriously not generated anymore. But I checked sourcegraph and the only place it is used is in its own test

File: src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java [hunk 2]
```diff
@@ -1,59 +0,0 @@
...
-        this.additionalProperties.put(name, value);
-    }
-
-    public Value withAdditionalProperty(String name, Object value) {
-        this.additionalProperties.put(name, value);
-        return this;
-    }
-
-    @Override
-    public int hashCode() {
```
Comment (bfink13) [inline src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java:43]: note that generated hashcode/equals/tostring no longer make use of commons-lang

File: src/test/java/com/indeed/rabbitmq/admin/RabbitManagementApiTest.java [hunk 3]
```diff
@@ -83,40 +82,6 @@ private static String toString(final Pattern pattern) {
         }
     }
 
-    private static boolean equals(final Permission actual, final Permission expected) {
```
Comment (bfink13) [inline src/test/java/com/indeed/rabbitmq/admin/RabbitManagementApiTest.java:86]: no clue why these were here. i switched to "native" equals (below)

File: src/main/resources/json/Bind.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     },
     "arguments": {
       "type": "object",
-      "javaType": "java.util.Map<String, Object>",
+      "existingJavaType": "java.util.Map<String, Object>",
```
Comment (jhamacher-indeed) [inline src/main/resources/json/Bind.json:12]: Why this change?

**Review by jhamacher-indeed — APPROVED** [review]

File: src/main/resources/json/Bind.json (hunk 4 above)
Comment (bfink13) [reply src/main/resources/json/Bind.json:12]: There was a change 10 months ago or something that split 'javaType' and 'existingJavaType' - anything that is a 'javaType' will be generated, and obviously we don't want to re-generate the Map class.

**Comment by bfink13** [comment]
Just force-pushed to force Travis to run

File: src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java (hunk 1 above)
Comment (ksitze) [reply src/main/java/com/indeed/rabbitmq/admin/pojo/Value.java:17]: I think this is a holdover from an earlier change. Should be fine without this.

File: src/test/java/com/indeed/rabbitmq/admin/RabbitManagementApiTest.java (hunk 3 above)
Comment (ksitze) [reply src/test/java/com/indeed/rabbitmq/admin/RabbitManagementApiTest.java:86]: Does the unit test run? I think there was some point where the `actual` argument could be `null`. If the unit test fails, then just use `Objects.equals()` instead.

**Review by mjpitz — COMMENTED** [review]
LGTM. What's used for generating hashcode and toString now?

**Comment by bfink13** [comment]
@mjpitz here's an example:
```
@Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append(Description.class.getName()).append('@').append(Integer.toHexString(System.identityHashCode(this))).append('[');
        sb.append("name");
        sb.append('=');
        sb.append(((this.name == null)?"<null>":this.name));
        sb.append(',');
        sb.append("description");
        sb.append('=');
        sb.append(((this.description == null)?"<null>":this.description));
        sb.append(',');
        sb.append("enabled");
        sb.append('=');
        sb.append(((this.enabled == null)?"<null>":this.enabled));
        sb.append(',');
        sb.append("additionalProperties");
        sb.append('=');
        sb.append(((this.additionalProperties == null)?"<null>":this.additionalProperties));
        sb.append(',');
        if (sb.charAt((sb.length()- 1)) == ',') {
            sb.setCharAt((sb.length()- 1), ']');
        } else {
            sb.append(']');
        }
        return sb.toString();
    }

    @Override
    public int hashCode() {
        int result = 1;
        result = ((result* 31)+((this.name == null)? 0 :this.name.hashCode()));
        result = ((result* 31)+((this.description == null)? 0 :this.description.hashCode()));
        result = ((result* 31)+((this.additionalProperties == null)? 0 :this.additionalProperties.hashCode()));
        result = ((result* 31)+((this.enabled == null)? 0 :this.enabled.hashCode()));
        return result;
    }

    @Override
    public boolean equals(Object other) {
        if (other == this) {
            return true;
        }
        if ((other instanceof Description) == false) {
            return false;
        }
        Description rhs = ((Description) other);
        return (((((this.name == rhs.name)||((this.name!= null)&&this.name.equals(rhs.name)))&&((this.description == rhs.description)||((this.description!= null)&&this.description.equals(rhs.description))))&&((this.additionalProperties == rhs.additionalProperties)||((this.additionalProperties!= null)&&this.additionalProperties.equals(rhs.additionalProperties))))&&((this.enabled == rhs.enabled)||((this.enabled!= null)&&this.enabled.equals(rhs.enabled))));
    }
```

**Comment by mjpitz** [comment]
Ok. Good to know. I think it might change how some things are logged, but nothing functional.
